examples/singleton_example/README.txt

```diff
@@ -0,0 +1,20 @@
+--- singleton_example Project ---
+
+ -- How to Build with cmake and make --
+   - cd to 'build' directory
```

Comment:
This can just be a code block of the commands.

Reply:
yes it can but given the whole structure of the readme I believe it is looking better this way for the eye